lib/Conversion/TritonToTritonGPU/TritonToTritonGPUPass.cpp

```diff
@@ -555,7 +557,13 @@ void populateTritonPatterns(TritonGPUTypeConverter &typeConverter,
       GenericOpPattern<triton::DotScaledOp>, GenericOpPattern<triton::CallOp>,
       TritonFuncOpPattern>(typeConverter, context);
 }
-
+// Proton patterns
+void populateProtonPatterns(TritonGPUTypeConverter &typeConverter,
+                            RewritePatternSet &patterns) {
+  MLIRContext *context = patterns.getContext();
+  patterns.add<GenericOpPattern<triton::proton::RecordOp>>(typeConverter,
```

Comment:
This can be taken out if we don't consider the frontend changes for now

Reply:
Maybe we don't need any conversion here anyway because proton's inputs are pointers but not tensors. Could you please double check? @CRobeck

Reply:
Scalars for now but could be tensors in the future. I don't think the conversion hurts anything here.